ocaml/middle_end/flambda/closure_conversion.ml

```diff
@@ -620,6 +622,7 @@ and close_functions t external_env function_declarations : Flambda.named =
          Misc.fatal_error "Closure_conversion: Tupled Alloc_local function found"
     in
     let params = List.mapi (fun i (v, kind) ->
+        (* CR ncourant: actually now we have the alloc_mode in lambda, propagate it *)
```

Comment:
This isn't totally trivial to propagate as `nheap` isn't determined solely be arg modes

Reply:
Ah no, this CR should remain here: it's about propagating the mode from `lparam` to flambda modes, instead of comparing the param position to `nheap` and assuming all trailing params are local.